File: padre/integration/test_files/test_prog.c
#include <stdio.h>

void func1();
void func2();
void func3();

void func1() {
    printf("Test 1\n");
    func2();
}

void func2() {
    func3();
}

void func3() {
    int a = 1;
    printf("Test %d\n", a);
}

int main() {
    func1();
    return 0;
}

File: padre/integration/test_files/test_prog.py
def c():
    return 'test string'


def d():
    return {
        "a": [1, 2, 3]
    }


def e():
    return d()


def a(b):
    print(c())
    print(b)
    print(e())
    return 456


print(a(123))
